CMIP6_studies/Ouranos.yaml

```diff
@@ -0,0 +1,355 @@
+- key: Ouranos tas bias
+  doi: [in-house performance]
+  metric: &tasclim
+    name: tasclim
+    long_name: tas climatology vs ERA5 climatology
+    units: K
+    variables: tas
+    comment: >
+      We calculate the RMSE at each grid point over the selected the domain (relative to ERA5) for the period from 1981 to 2010 and then we do a spatial average.
+      All models and ERA5 are linearly interpolated on a regular 1°x1° grid.  
+      -----
+      Periods, domains can be changed in https://pavics.ouranos.ca/selection-de-modeles
+    best: 1.21
+    worst: 2.26
+  type: performance
+  spatial_scope: NAM-CORDEX
+  temporal_scope: Climatology
+  period:
+    reference: 1981-2010
+  plausible_values:
+    min: 0
+    max: +inf 
+    source: Ouranos
+    comment: > 
+      
+  data_source: author
+  data:
+    EC-Earth3-Veg: 1.21
+    CanESM5-1-EBC: 1.32
+    CESM2: 1.38
+    MPI-ESM1-2-HR: 1.41
+    IPSL-CM6A-LR: 1.45
+    CMCC-ESM2: 1.49
+    CNRM-ESM2-1: 1.523955
+    EC-Earth3: 1.55
+    CNRM-CM6-1:	1.60
+    NorESM2-MM:	1.61
+    NorESM2-LM:	1.62
+    TaiESM1: 1.63
+    CMCC-CM2-SR5: 1.65
+    ACCESS-CM2:	1.66
+    MPI-ESM1-2-LR: 1.67
+    ACCESS-ESM1-5: 1.68
+    CNRM-CM6-1-HR: 1.89
+    UKESM1-0-LL: 1.96
+    MIROC6:	2.04
+    CanESM5-1: 2.11
+    MIROC-ES2L:	2.26
+
+- key: Ouranos hu-850mb bias
+  doi: [in-house performance]
+  metric: &hu850clim
+    name: huclim-850hPa
+    long_name: Specific humidity at 850-mb climatology vs ERA5 climatology
+    units: kg/kg
+    variables: hu-850-mb
+    comment: >
+      We calculate the RMSE at each grid point over the selected the domain (relative to ERA5) for the period from 1981 to 2010 and then we do a spatial average.
+      All models and ERA5 are linearly interpolated on a regular 1°x1° grid.  
+      -----
+      Periods, domains can be changed in https://pavics.ouranos.ca/selection-de-modeles
+    best: 0.000274
+    worst: 0.000887
+  type: performance
+  spatial_scope: NAM-CORDEX
+  temporal_scope: Climatology
+  period:
+    reference: 1981-2010
+  plausible_values:
+    min: 0
+    max: +inf 
+    source: Ouranos_SAC
```

Comment:
`source` values come from a controlled vocabulary
https://github.com/WCRP-CORDEX/cmip6-for-cordex/blob/e6056efdbb698d17fcbd412db61cd27b43243ca9/CMIP6_studies/README.md?plain=1#L90
If you need to provide details, you can use the comment